engine/flink/kafka-util/src/main/scala/pl/touk/nussknacker/engine/kafka/KafkaSinkFactory.scala

```diff
@@ -29,8 +32,10 @@ class KafkaSinkFactory(config: KafkaConfig,
              @NotBlank
              topic: String
             )(metaData: MetaData): Sink = {
-    val serializationSchema = schemaFactory.create(topic, config)
-    new KafkaSink(topic, serializationSchema, s"${metaData.id}-$topic")
+    val objectNaming = objectNamingProvider.create(getClass.getClassLoader)
```

Comment:
I wonder if it is a good place to invoke `create()` and use service loader. We usually trying to keep it on very high level like in `ModelData` or `CompiledProcess.compile` just to make sure that we have separation of service loading (where many strange failures regards to classloaders can happen) and simple creation of objects. Maybe we should pass `ObjectNaming` instead of `ObjectNamingProvider` to `KafkaSourceFactory` etc?